routes/definitions.js

```diff
@@ -17,12 +17,22 @@ async function getDefinition(request, response) {
   const coordinates = utils.toEntityCoordinatesFromRequest(request)
   const pr = request.params.pr
   // if running on localhost, allow a force arg for testing without webhooks to invalidate the caches
-  const force = request.hostname.includes('localhost') ? request.query.force || false : false
+  const force = (request.hostname && request.hostname.includes('localhost')) ? request.query.force || false : false
   const expand = request.query.expand === '-files' ? '-files' : null // only support '-files' for now
   const result = await definitionService.get(coordinates, pr, force, expand)
+  if (Array.isArray(result.data) && result.data.length > 0) {
+    response.set('cache-tag', definitionService.tagFromCoordinates(result.data))
+  }
```

Comment:
in this case result is a definition object - there is no Array or data property here, it is always a single one

```
result == {
 described: { ... },
 licensed: { ... },
 ...
}
```

Reply:
Thanks for catching this. Incorporated at https://github.com/clearlydefined/service/pull/503/commits/796203435c418209592871e49ea0f592b65aeaed